uui-core/src/hooks/useLayer.ts

```diff
@@ -0,0 +1,13 @@
+import { useEffect } from 'react';
+import { useUuiContext } from '../services';
+
+export const useLayer = () => {
+    const context = useUuiContext();
+    const layer = context.uuiLayout?.getLayer();
```

Comment:
you will get new layer on each hook call, it's better to use useRef hook to avoid that.